sdk/resourcemanager/Azure.ResourceManager.Core/samples/Sample3_CreatingAVirtualNetwork.md

```diff
@@ -0,0 +1,36 @@
+Example: Creating a Virtual Network
+--------------------------------------
+
+In this example we will be create a VirtualNetwork.  Since the SDK follows the resource
+hierarchy in Azure, we will need to do this inside of a ResourceGroup.  We will start by creating
+a new resource group like we did above
```

Comment:
```suggestion
In this example, we'll create a virtual network. Since the SDK follows the resource hierarchy in Azure, we'll need to do this inside of a resource group. Start by creating a new resource group, like we did above:
```